VideoWeb/VideoWeb.EventHub/Hub/IEventHubClient.cs

```diff
@@ -30,5 +30,6 @@ public interface IEventHubClient
         Task ParticipantsUpdatedMessage(Guid conferenceId, List<ParticipantResponse> participants);
         Task HearingLayoutChanged(Guid conferenceId, Guid changedById, HearingLayout newLayout, HearingLayout oldLayout);
         Task NewConferenceAddedMessage(Guid conferenceId);
+        Task AllocationHearings(string csoUserName, IList<HearingDetailRequest> hearings);
```

Comment:
Could you make this `List<HearingDetailRequest>` like line 30 for consistency?

Reply:
done